pkg/templates/csisidecars.go

```diff
@@ -91,6 +91,7 @@ var SnapshotterContainer = corev1.Container{
 		"--timeout=150s",
 		"--leader-election=true",
 		"--extra-create-metadata=true",
+		"--enable-volume-group-snapshots=true",
```

Comment:
This flag doesnt exists on all snapshotter sidecar versions. this exists in only ODF 4.16 and ODF 4.17 not below that and also if the CRD's doesn't exists we should not set this to true, because the snapshotter controller will crash.